plugins/sharding/stream_shard_executor.rb

```diff
@@ -43,6 +43,28 @@ def have_record?
         true
       end
 
+      protected
```

Comment:
Why is it needed?
Generally, `protected` isn't used in Ruby.

Reply:
> Generally, protected isn't used in Ruby.

I see. (I didn't know that.)

I have removed this method access modifier.